Adding one ECCV 2022 paper (XDED)

leekyungmoon opened the pull request:
Hi, thank you for your great repository.

We respectfully request to add our ECCV 2022 paper "_Cross-Domain Ensemble Distillation for Domain Generalization_" (XDED).

We would be grateful if you could approve our request.

Best,
Kyungmoon

Commit:
adding one ECCV 2022 paper (XDED)

amber0309 commented:
Hi leekyungmoon,

Thank you for your PR! Since I just merged an earlier PR, which induces some confilict between your PR and branch master. Could you please kindly update and resubmit your PR?  :)

leekyungmoon committed:
Merge branch 'master' into xded

leekyungmoon commented:
Hi, @amber0309. Thanks for pointing that out.

I removed the conflicts and moved the ECCV'22 paper to '## Research papers 2022-Computer vision venues!'

Could you please merge my updated PR? Thanks again for this great repository!

Best, Kyungmoon
